### ISSUE TITLE
Image normalization description doesn't match code

### ISSUE THREAD

**Issue by pjl54** [MouseLand/cellpose#469]
The normalization method used in models.eval describes the normalization of images to be between 0 and 1.0:

https://github.com/MouseLand/cellpose/blob/706bae639a30cdd0a5e4966f33f28da08a200b4d/cellpose/models.py#L159

But I believe the code is calling this function:

https://github.com/MouseLand/cellpose/blob/706bae639a30cdd0a5e4966f33f28da08a200b4d/cellpose/transforms.py#L194

For an image with a max value of 15, a min value of 1, a 1st percentile value of 2, and a 99th percentile value of 10, this would normalize between (1-2)/(10-2) = -0.125 and (15-2)/(10-2) = 1.625.

**Comment by kevinjohncutler** [comment]
@pjl54, that's right. The Omnipose version of this function fixes this issue. The 1-99 percentile range can be problematic itself (issue #454) but the default cellpose implementation giving negative numbers is not good for any code that assumes nonnegative image data (like applying a gamma <1).

**Comment by carsen-stringer** [comment]
many packages use mean-zero centered images (default normalization for imagenet for example), there's no problem in doing this in neural networks. in the docs it says that

```
normalize: bool (optional, default True)
            normalize data so 0.0=1st percentile and 1.0=99th percentile of image intensities in each channel
```

this is not between 0.0 and 1.0 as you've described, can you please tell me where the faulty documentation is?

**Comment by kevinjohncutler** [comment]
@carsen-stringer I think what is unclear to @pjl54 is whether or not the image is supposed to be clipped so that the end image lies in the range 0-1. The most straightforward reading of the documentation is consistent with the code, meaning that the value range gets shifted and scaled such that the 1st percentile lands at 0 and the 99th lands at 1. So values below the 1st percentile become negative and the values above the 99th percentile become greater than 1. This is completely clear as-is, but maybe the confusion comes from what users typically expect from normalization, where we know exactly what the minimum and maximum values will be a priori (0 and 1).
